app/views/home/index.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="home__image-wrap">
   <div class="home__image"></div>
 </div>
+<div id="banner"><%=t('.mfa_banner_html')%></div>
```

Comment:
If we plan on using this `id` for other purposes beyond styling (e.g. to link to this page with this element anchored to the top), then we can keep the `id` for _that_ purpose. Otherwise, I'd change the `id` to a `class`.

That said, if we plan on using the `id` for other purposes, let's keep the `id` _and_ add a `class` for styling-specific conerns.